distribution/neumann_alloc.json

```diff
@@ -1 +1 @@
PoghkdiqF6cYdEg15GjsnHwWgruocQg4fkRr",4000000000000],["6BKgiWmXEbtX1ehKJ2ZV3tVbx65hfzMEDKPq3Lk4zKE2wtNN",1000000000000],["6BKm2kN8mgQ8eoNSicZwYHWHjupisv4FYuQSBGxaSjF59c79",1000000000000],["6BKV9WJS91SBrBmLHZc1hwuJZoLQ7vUTTtTRHNXe2CABZD1A",1000000000000],["6BmWtzVr4zCScZPoJCwkXxjvcdRUdJk7JArUa1TSyTmAuCRP",3000000000000],["6BmYr7DZ4EDTNQgovqM1b3eNRiL6kLkrxyRvoYqkVXQmvpga",1000000000000],["6BmAufYcVJLyCWhhMyg9zMRWCNHdT14u3jV2d4Zzw1pz3AkG",1000000000000],["6BMq33rc4N8T1YD4fBo3mTg4kkgNSEQqon6RQecoiaVunuqk",2000000000000],["6BMQWuqEqEzM65EXjsKytogawPDPNfAdtrnZGNKLmtCk1vKv",1000000000000],["6BmTNQSBJx5qf9TTDzckdXg2FgRbPrSVkQAs8htTEABj5MHk",1000000000000],["6Bp4g338xTZXRYGjAVi9G1zQbvEmUPR7TKhT8ZFubofUwYDu",999853883957],["6BpU3KK7bURFi3ExJASgYgmUp9vzRNPEbGKZRGYnnB7Vgmuf",1000000000000],["6BQdeHEiV82gcW4uwVNg7SzFZR5KeWkHBCXA9bVML6zeauay",1000000000000],["6BQECuUSNa8KtsJHpe6HwPwGRsYWXfKLcdgPLCkTa64aUG5w",1000000000000],["6BQppM61mqcVJaWZsnEjZFzkQQzaKkFC4QqfLMwFQ8hSSGFe",1000000000000],["6BSYnz29Pd8GT1JoZ9YUBHP1b884DRwtckz1iQEWUq86dQGe",3000000000000],["6BrbMcQQVQQ8mncUhjrN7VSUgwYcj1JDcJa78DS6Wm5c3GN8",1000000000000],["6BSgebDQ7DeJ2c27d7zAAbpxrnnoQUfZLSykqW3eZHFYhjxm",1000000000000],["6BrQYuqfusefxVheVfTPWV33ysEBocRtojCZEm4KyuEmBvWa",1000000000000],["6BSVDkpoJ9cmY3VsFEqWDzxqMjNTtAzYS7wKSEkk1yNM7vMZ",1000000000000],["6BT8Qtg4RoqBE58GJjpSAMTGFzn6HioBiSNc4maMN3ueZUeg",1000000000000],["6BsAoBZtRyiS8G342uzXaynFQjwcCohMMR4Brcf8jzcjjYny",1000000000000],["6BsEcZxZCEH2GP1gzHHnE2xW6XDRV9TjsJ7HEVD7PbdWmo66",3000000000000],["6BTkz5rGFe4ZrmpcATmo6X3C2ruuX3pry5jTQLA386BSbEzq",1000000000000],["6BTRgKpfNw74DbhJgt5YjtUD9k1JPYWa8DtTqomGwHVP7v2y",1000000000000],["6BTToUkaSxpgLZXMr8q8ykYQGbvcWoALGfVk77NEqT8Y6DjW",1000000000000],["6BtgVQt59vDPgMzdFYvdQHkwoG763KUtjmbERWUUcfwjLJmG",1000000000000],["6BtnEa9Sp3XXMwU998xca7ahbtJYH3Bbke1dydWwpainQQom",1000000000000],["6BuqVTpNn8E1hmhepsoQw72wHtnCF4i1wcUMC6iJ4zuD7eBW",1000000000000],["6BV9zo11eXZWn5Ekkc48HcQxfHuP72XrzVw7LEjcq33Gunt8",1000000000000],["6BVZbDozBiXppykzgfZXLZYKGJvMWPvHwNxQzKazuHxxNt5B",1000000000000],["6BVD1No41w9MG13yWNrxfJdaudCbQgNjS456Nhi4g2JRWy8A",1000000000000],["6BVg9HcRSBiZXvWR5T2jChyzJ43nFN2Kqkd2o27k6wmpQxea",1000000000000],["6BVtWqy24u79j1vG3uic1FKp3rY6EZNuhqR7Mx5491Aj84wn",1000000000000],["6BWGKaRNRjLaMNkYXLdPTK69XDRUyaUGTbi6oMyKMzZi6oZD",1000000000000],["6BWPN6kLRD69QQwLY5rHCh18yJAMTBs3fenhSqLcedqK69DP",1000000000000],["6BXinvBPdGfoNxedxFzZjpseFdytSRPM1fmWBSVMUsKsmmr7",2000000000000],["6BXtRDCvjrqud1TNQr5hCFFiPxt4qJZaRYDfktPqJwNrRkEL",3000000000000],["6BY7Yd2YyWZ6b7RuVa2xgvjq4xqWDWM7L8uUbvevRVzHCk3z",1000000000000],["6BYYuJCEuQbbsopuMyAAnveQBKP9DGYXjw1zQaeAuVdzpo51",1000000000000],["6BYwnT8mYhLxCd19FvdShcnYjEH3idQGRDgfJ8Qm9i5o5PLb",1000000000000],["6Bb4g5Bj4ThiyDpWCiSeVuxwe9cRzdfs8Bq3dKyirMLJNLSq",3000000000000],["6Bb5bf8daBfF1VrrnaKxDb3jXei3cZ3teoroenokvmHy5C3x",1000000000000],["6BbHWy6Gib6HoUR7yJrTjs8k6sZ2bXXktqyW8T4Xm2mczitX",1000000000000],["6BdG7xJaegxjZ9z3s3mPsrCADxCGqo3ucYtMvmoA2fZa6QXG",1000000000000]]
```

Comment:
the current neumann_vesting json relies on the following balance for this account: ["66i43PKEDXJNpqaHHop4aaxEeUPCZ93vcQxVURcPfko6XQFa",3133333765840238435].

however, idk why it doesn't seem to work with neumann_alloc in unit tests, so i swapped back to:
["66i43PKEDXJNpqaHHop4aaxEeUPCZ93vcQxVURcPfko6XQFa",3233333765840238435].